### PR TITLE
Iron out remaining issues in batch date edit

### PR DESCRIPTION
**Pull request by Coenego**
- [x] When an empty term is selected and any week (except 1) is clicked, the new event is always added to week 1. This doesn't happen when an event was already present.

- [x] When an event has been added on a date, the user clicks 'add another day' and selects that same day of the week, that event gets duplicated in the list. We should avoid adding 2 events on the same day of the same week through batch edit.

This is part of followup for https://github.com/CUL-DigitalServices/grasshopper-ui/pull/343

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

**Comment by bp323** [comment]
One remaining issue. When starting from an empty term and adding an event in week 1 by clicking the week, then proceeding to click week 2, an error is shown in the console: `Uncaught Error: An invalid value for date has been provided`

File: shared/gh/js/views/gh.admin-batch-edit-date.js [hunk 1]
```diff
@@ -93,6 +93,29 @@ define(['lodash', 'moment', 'gh.core', 'gh.api.config'], function(_, moment, gh,
     };
 
     /**
+     * Disable the already chosen days in the date pickers
+     *
+     * @private
+     */
+    var updateDayPickers = function() {
```
Comment (bp323) [inline shared/gh/js/views/gh.admin-batch-edit-date.js]: All it does is disabling so maybe we can rename to `disableDayPickers`

**Comment by bp323** [comment]
Back to @Coenego for followup

**Comment by Coenego** [comment]
Back to @bp323 for review after follow-up

**Comment by Coenego** [comment]
Assigning to @bp323 for review

**Comment by bp323** [comment]
Issue remaining: When adding events through batch edit for a few weeks, then changing the start/end times for them in batch edit and adding an extra week, the events are duplicated.

**Comment by Coenego** [comment]
Back to @bp323 for review

**Comment by bp323** [comment]
Verified and merged.

**Pull request merged**